project/Build.scala

```diff
@@ -943,6 +875,7 @@ object Build {
       mimaBinaryIssueFilters ++= BinaryIncompatibilities.SbtPlugin,
 
       addSbtPlugin("org.portable-scala" % "sbt-platform-deps" % "1.0.0"),
+      libraryDependencies += "org.scala-js" %% "scalajs-env-nodejs" % "1.1.1",
```

Comment:
Add `"org.scala-js" %% "scalajs-js-envs" % "1.1.1"` as well? Better not to rely on transitive dependencies...